API/app.js

```diff
@@ -12,6 +12,8 @@ app.get("/api", getAPI);
 
 app.get("/api/articles/:article_id", getArticle);
 
+app.get("/get/articles", getArticles);
```

Comment:
I'm very aware that this should be "/API/articles" and it was when I committed. Because the tests wouldn't otherwise pass? This and my endpoints.json seem to have not been pushed properly, but I've only noticed after pulling the main branch down.